app/controllers/application_controller.rb

```diff
@@ -1,3 +1,11 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
   def index; end
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name posts_counter])
```

Comment:
I think you missed the `posts_counter` variable from the last project here 😅

Reply:
Sorry for that. I will fix it! 🤦‍♂️